src/Tools/dotnet-user-jwts/src/Commands/CreateCommand.cs

```diff
@@ -0,0 +1,204 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Globalization;
+using System.Linq;
+using Microsoft.Extensions.CommandLineUtils;
+using Microsoft.Extensions.Tools.Internal;
+
+namespace Microsoft.AspNetCore.Authentication.JwtBearer.Tools;
+
+internal sealed class CreateCommand
+{
+    private static readonly string[] _dateTimeFormats = new[] {
+        "yyyy-MM-dd", "yyyy-MM-dd HH:mm", "yyyy/MM/dd", "yyyy/MM/dd HH:mm" };
+    private static readonly string[] _timeSpanFormats = new[] {
+        @"d\dh\hm\ms\s", @"d\dh\hm\m", @"d\dh\h", @"d\d",
+        @"h\hm\ms\s", @"h\hm\m", @"h\h",
+        @"m\ms\s", @"m\m",
+        @"s\s"
+    };
+
+    public static void Register(ProjectCommandLineApplication app)
+    {
+        app.Command("create", cmd =>
+        {
+            cmd.Description = "Issue a new JSON Web Token";
+
+            var schemeNameOption = cmd.Option(
+                "--scheme",
+                "The scheme name to use for the generated token. Defaults to 'Bearer'",
+                CommandOptionType.SingleValue
+                );
+
+            var nameOption = cmd.Option(
+                "--name",
+                "The name of the user to create the JWT for. Defaults to the current environment user.",
+                CommandOptionType.SingleValue);
+
+            var audienceOption = cmd.Option(
+                "--audience",
+                "The audiences to create the JWT for. Defaults to the URLs configured in the project's launchSettings.json",
+                CommandOptionType.MultipleValue);
+
+            var issuerOption = cmd.Option(
+                "--issuer",
+                "The issuer of the JWT. Defaults to the dotnet-user-jwts",
+                CommandOptionType.SingleValue);
+
+            var scopesOption = cmd.Option(
+                "--scope",
+                "A scope claim to add to the JWT. Specify once for each scope.",
+                CommandOptionType.MultipleValue);
+
+            var rolesOption = cmd.Option(
+                "--role",
+                "A role claim to add to the JWT. Specify once for each role",
+                CommandOptionType.MultipleValue);
+
+            var claimsOption = cmd.Option(
+                "--claim",
+                "Claims to add to the JWT. Specify once for each claim in the format \"name=value\"",
+                CommandOptionType.MultipleValue);
+
+            var notBeforeOption = cmd.Option(
+                "--not-before",
+                @"The UTC date & time the JWT should not be valid before in the format 'yyyy-MM-dd [[HH:mm[[:ss]]]]'. Defaults to the date & time the JWT is created",
```

Comment:
```suggestion
                @"The UTC date & time the JWT should not be valid before in the format 'yyyy-MM-dd [[HH:mm[[:ss]]]]'. Defaults to the date & time the JWT is created.",
```

Reply:
This one still seems to have the overly complex datetime format description still.